### PR TITLE
addedfunctionaltestinnightlypipeline

### PR DESCRIPTION
**Pull request by sivas23**
**Before creating a pull request make sure that:**

### Change description ###
Added functional tests in nightly pipeline

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -6,17 +6,25 @@ properties([
 
 @Library("Infrastructure")
 
-def type = "java"
+def type = "nodejs"
```
Comment (geek-soft) [inline Jenkinsfile_nightly]: Revert to Java and run functional tests from Gradle.

Comment (sivas23) [reply Jenkinsfile_nightly]: Reverted to Java and added command to run functional tests from Gradle

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -12,11 +12,20 @@ def product = "fpl"
...
+//  enableMutationTest()
+//
+//  after('mutationTest') {
+//    withSonarQubeEnv("SonarQube") {
+//      sh "./gradlew --info sonarqube"
+//    }
+//  }
+
+  //to Run functional test
+  sh "./gradlew runFunctionalTest"
```
Comment (geek-soft) [inline Jenkinsfile_nightly]: I do not think it would even run TBH. It is definite not what the pipeline should look like.

Comment (geek-soft) [reply Jenkinsfile_nightly]: Also calling `runFunctionalTest` itself would not work as there would not be yarn dependencies installed. A `functional` task is an entry point to functional tests.

Comment (sivas23) [reply Jenkinsfile_nightly]: Hi Damian, changed to functional task as an entry point.

**Comment by geek-soft** [comment]
Closing to start from scratch.

**Pull request closed without merging**